plugins/CoreAdminHome/lang/en.json

```diff
@@ -108,6 +109,24 @@
         "MissingTrackingCodeEmailSubject": "No traffic for %s recorded in Matomo Analytics, get started now",
         "JsTrackingCodeMissingEmail1": "A few days ago you added the website '%s' to your Matomo Analytics. We just checked and your Matomo doesn't seem to have any recorded traffic for this website.",
         "JsTrackingCodeMissingEmail2": "To begin tracking data and getting insights into your users, you'll need to setup tracking in your website or mobile app. For websites simply embed the tracking code right before the %s tag.",
-        "JsTrackingCodeMissingEmail3": "To find and customize your tracking code, %1$sclick here%2$s (or have a look at the %3$sJavaScript Tracking Client guide%4$s)."
+        "JsTrackingCodeMissingEmail3": "To find and customize your tracking code, %1$sclick here%2$s (or have a look at the %3$sJavaScript Tracking Client guide%4$s).",
+        "TrackingFailuresIntroduction": "This page lists tracking failures that happened during the last %s days. Please note that only the most common kind of tracking failures are recorded and not all of them.",
+        "NoKnownFailures": "There are no known tracking failures.",
+        "Problem": "Problem",
+        "Solution": "Solution",
+        "TrackingURL": "Tracking URL",
+        "LearnMore": "Learn more",
+        "DeleteAllFailures": "Delete all failures",
+        "NTrackingFailures": "%s tracking failures",
+        "ViewAllTrackingFailures": "View all tracking failures",
+        "TrackingFailureInvalidSiteProblem": "The site does not exist.",
+        "TrackingFailureInvalidSiteSolution": "Update the configured idSite in the tracker.",
+        "TrackingFailureAuthenticationProblem": "Request was not authenticated but should have.",
```

Comment:
Would suggest a change to `Request was not authenticated but authentication was required.` (if I am understanding the error correctly)

Reply:
Will update it 👍